### PR TITLE
Improved Code Formatting + Shell Legacy Syntax Removed

### PR DESCRIPTION
**Pull request by harens**
The PR does not necessarily fix any existing issues, but instead works on improving syntax.

To achieve this, [Align-YAML](https://github.com/jonschlinkert/align-yaml), [Beautysh](https://github.com/bemeurer/beautysh), [Uncrustify](https://github.com/uncrustify/uncrustify), and [ShellCheck](https://github.com/koalaman/shellcheck) have been used.
The [octoStyle.cfg](https://github.com/harens/Ocquarium/blob/master/octoStyle.cfg) file is used to configure Uncrustify, and is based on [ben.cfg.txt](https://github.com/uncrustify/uncrustify/blob/master/documentation/htdocs/ben.cfg.txt).

[Dependabot](https://dependabot.com/) updated dependencies, and [ImgBot](https://imgbot.net/) optimised images.

### CODE REVIEW

File: ic_launcher.svg [hunk 1]
```diff
@@ -1,9 +1 @@
...
-  <path d="M24.0,24.0m-14.0,0.0a14.0,14.0,0,1,1,28.0,0.0a14.0,14.0,0,1,1,-28.0,0.0" fill="#5c6bc0" />
-  <path d="M37.7829445,26.469236 L29.6578482,18.3441397 L18.3441397,29.6578482 L26.469236,37.7829445 C29.1911841,37.2979273 31.7972024,36.0037754 33.9004889,33.9004889 C36.0037754,31.7972024 37.2979273,29.1911841 37.7829445,26.469236 Z" fill="#3f51b5" />
-  <path d="M24.0,24.0m-8.0,0.0a8.0,8.0,0,1,1,16.0,0.0a8.0,8.0,0,1,1,-16.0,0.0" fill="#FFFFFF" />
-</svg>
\ No newline at end of file
+<svg height="216" viewBox="0 0 48 48" width="216" xmlns="http://www.w3.org/2000/svg"><path d="m44 24.2010101-10.0995111-10.0995111-19.7989899 19.7989899 50.0995111 50.0995111c-14.9484297-20.0502071-9.9122537-22.0024973-6.0578805-25.8568704 3.8543731-3.8543732 5.8066633-8.8905492 5.8568704-13.9421195z" fill="#1a237e"/><path d="m10 24a14 14 0 1 1 28 0 14 14 0 1 1 -28 0" fill="#5c6bc0"/><path d="m37.7829445 26.469236-8.1250963-8.1250963-11.3137085 11.3137085 8.1250963 8.1250963c2.7219481-.4850172 5.3279664-1.7791691 7.4312529-3.8824556s3.3974384-4.7093048 3.8824556-7.4312529z" fill="#3f51b5"/><path d="m16 24a8 8 0 1 1 16 0 8 8 0 1 1 -16 0" fill="#fff"/></svg>
```
Comment (ZeevoX) [inline ic_launcher.svg]: The Screenshot_N.png files, art.png, and svg files compressed in this commit are not used and not packaged into the APK. art-min.png is an already-compressed and optimized png, which is used in README.md, thus this commit is removing readability from SVG files unnecessarily.

**Comment by ZeevoX** [comment]
Do Uncrustify, ImgBot, Dependabot, BeautySH and Align-YAML run automatically? How do these tools work?

**Comment by harens** [comment]
Uncrustify works by specifying a configuration file, an input file and an output file. For Ocquarium,  the configuration file is [octoStyle.cfg](https://github.com/harens/Ocquarium/blob/master/octoStyle.cfg), whilst the input file would be the same as the output file, and could be any Java file _e.g. SettingsActivityV2.java_

So one example of a valid command would be:
`uncrustify -c octoStyle.cfg -f SettingsActivity.java -o SettingsActivity.java`

You can find out more about running Uncrustify [here](https://github.com/uncrustify/uncrustify#running-the-program)

ImgBot and Dependabot can both be found from [GitHub Marketplace](https://github.com/marketplace)

ImgBot is automatic by default, though can be [configured](https://imgbot.net/docs#configuration)

Dependabot needs to be told which repos to manage and what language they're in. For Java, you have to specify whether the project uses Maven or Gradle (and it's in beta for Java)

BeautySH can be configured through [flags](https://github.com/bemeurer/beautysh#usage) and special comments. It is not automatic, but can beautify in bulk very well.

Finally Align-YAML doesn't necessarily require any arguments at all. However, for your project, I simply ran `align .travis.yml`. You can find out more about it [here](https://github.com/jonschlinkert/align-yaml#usage)

I hope that's answered your second question. To answer your first question, only the bots can be easily configured to be automatic. However, I would be more than happy to write a shell script to autolint your project.

I hope this helped!

File: ic_launcher.svg (hunk 1 above)
Comment (harens) [reply ic_launcher.svg]: Sorry about that. I have reverted the relevant images back to their original state

File: ic_launcher.svg [hunk 2]
```diff
@@ -1 +1,9 @@
-<svg height="216" viewBox="0 0 48 48" width="216" xmlns="http://www.w3.org/2000/svg"><path d="m44 24.2010101-10.0995111-10.0995111-19.7989899 19.7989899 50.0995111 50.0995111c-14.9484297-20.0502071-9.9122537-22.0024973-6.0578805-25.8568704 3.8543731-3.8543732 5.8066633-8.8905492 5.8568704-13.9421195z" fill="#1a237e"/><path d="m10 24a14 14 0 1 1 28 0 14 14 0 1 1 -28 0" fill="#5c6bc0"/><path d="m37.7829445 26.469236-8.1250963-8.1250963-11.3137085 11.3137085 8.1250963 8.1250963c2.7219481-.4850172 5.3279664-1.7791691 7.4312529-3.8824556s3.3974384-4.7093048 3.8824556-7.4312529z" fill="#3f51b5"/><path d="m16 24a8 8 0 1 1 16 0 8 8 0 1 1 -16 0" fill="#fff"/></svg>
\ No newline at end of file
+<?xml version="1.0" standalone="no"?>
```
Comment (ZeevoX) [inline ic_launcher.svg]: Yay! thanks

File: app/src/main/java/com/zeevox/octo/settings/SettingsActivityV2.java [hunk 3]
```diff
@@ -691,18 +691,5 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
          listView.setDivider(new ColorDrawable(Color.TRANSPARENT));
          listView.setDividerHeight(0);
       }
-
```
Comment (ZeevoX) [inline app/src/main/java/com/zeevox/octo/settings/SettingsActivityV2.java]: This is here on purpose, I can explain why.

**Comment by harens** [comment]
By the looks of it, it's not easy to configure Uncrustify to work with Google's Java Style. Instead, I'll work on it with google-java-format

**Comment by ZeevoX** [comment]
> By the looks of it, it's not easy to configure Uncrustify to work with Google's Java Style. Instead, I'll work on it with google-java-format

That's awesome! Thank you very much.

**Comment by ZeevoX** [comment]
I have cherrypicked the following commits so far:

https://github.com/ZeevoX/Ocquarium/commit/b2bd94bbbc8c1782c5e308a1ded905d9c077e133
https://github.com/ZeevoX/Ocquarium/commit/5345c001bac5ce1e64614637aa2c1eef7c86c941
https://github.com/ZeevoX/Ocquarium/commit/fb4de44df757bc2a9c80b52e69b6f7a40a60ecae
https://github.com/ZeevoX/Ocquarium/commit/f1200c3223ff4635bd044c15e59c83b9b0e224b9
https://github.com/ZeevoX/Ocquarium/commit/31b4f5047a53de04c21259781e142d417449a5d5
https://github.com/ZeevoX/Ocquarium/commit/05bca2a9429966a238dbd4d52c67b080c2fd482f
https://github.com/ZeevoX/Ocquarium/commit/f02cff4db098d6da612c120145d472eebcdc8743
https://github.com/ZeevoX/Ocquarium/commit/799ccaf6a3c599e5d62802542afe18db6c1bff89
https://github.com/ZeevoX/Ocquarium/commit/5338cd7bbbd1ac2d1b51431128f0e98993b4512f
https://github.com/ZeevoX/Ocquarium/commit/12490fe599b5fb40b481a5a37a5e90ea4d779016
https://github.com/ZeevoX/Ocquarium/commit/840478c54deee0c8f99dd1dce969c8fb0118dbf5

Thank you for your work, @harens!

**Comment by harens** [comment]
Just to make things a little cleaner, I'll refork Ocquarium, and push it as a seperate pull request

**Pull request closed without merging**